src/scripts/app.ts

```diff
@@ -71,51 +70,57 @@ async function checkChosedValue() {
 
 async function fetchData(chosedValue: number) {
 	try {
-		unsplash.search
-			.getPhotos({ query: 'star-wars', orientation: 'portrait', perPage: 30 })
-			.then((result) => {
-				if (!result.response?.total) {
-					showToast('error', 'Cannot get API data');
-				} else {
-					result.response.results.forEach((item) => imageArray.push(item));
-					shuffleArray(imageArray);
-					addItems(chosedValue, imageArray);
-				}
-			});
+		const data = await unsplash.search.getPhotos({
+			query: 'star-wars',
+			orientation: 'portrait',
+			perPage: 30,
+		});
+
+		if (!data.response?.results) {
+			showToast('error', 'Cannot get API data');
+		} else {
+			data.response?.results.forEach((item) => fetchArray.push(item));
+			shuffleArray(fetchArray);
+			addItemsToGameGrid(chosedValue, fetchArray);
+		}
 	} catch (error) {
 		showToast('error', 'Data fetch failure');
 	}
 }
 
-function shuffleArray(imageArray: Basic[]) {
-	for (let i = 0; i < imageArray.length; i++) {
-		const random = Math.floor(Math.random() * imageArray.length);
-		[imageArray[i], imageArray[random]] = [imageArray[random], imageArray[i]];
+function shuffleArray(fetchArray: Basic[]) {
+	for (let i = 0; i < fetchArray.length; i++) {
+		const random = Math.floor(Math.random() * fetchArray.length);
+		[fetchArray[i], fetchArray[random]] = [fetchArray[random], fetchArray[i]];
 	}
 }
 
-function addItems(chosedValue: number, imageArray: Basic[]) {
-	const selectedImages = imageArray.slice(0, chosedValue);
+function addItemsToGameGrid(chosedValue: number, fetchArray: Basic[]) {
+	const selectedImages = fetchArray.slice(0, chosedValue);
 
 	if (selectedImages) {
 		selectedImages.forEach((item: Basic, i: number) => {
 			const imgItem = document.createElement('img');
-			imgItem.classList.add('js-game-img');
+			imgItem.classList.add('game__image');
 			imgItem.id = `${i++}`;
 			const imgSource = item.urls.small;
 			imgItem.src = imgSource;
 
 			imgItemsArray.push(imgItem);
 			gameGrid.appendChild(imgItem);
+
+			setTimeout(() => {
+				imgItem.classList.add('hidden');
+			}, 1700);
```

Comment:
A ten czas nie powinien zależeć od levelu? Hard to Ci mignie, medium to moze byc 2s, a easy to np. 3s